chsdi/models/bod.py

```diff
@@ -52,6 +52,7 @@ def layerMetadata(self):
         primaryAttrs = ('layerBodId', 'idGeoCat', 'name', 'fullName')
         excludedAttrs = ('staging', 'searchUnicode', 'chargeable')
         meta = {'attributes': {}}
+        # 2to3 would wrap self.__dict__.keys() in a list(), which is not necessary in this case
```

Comment:
dito remove comment

Reply:
A thought on all those comments: Probably they do make sense. When someone in future wants to port the code to Python3 and uses 2to3 all those things will pop up. And in places, where I added this comment, I already tested it and checked, that the suggested changes of 2to3 are not needed. So probably re-adding those comments make sense.